src/main/java/com/bgsoftware/superiorskyblock/core/menu/button/impl/menu/BiomeButton.java

```diff
@@ -34,6 +35,8 @@ public class BiomeButton extends SuperiorMenuButton<MenuBiomes> {
     private final TemplateItem lackPermissionItem;
     private final List<String> lackPermissionCommands;
     private final Biome biome;
+    private final YamlConfiguration cfg;
```

Comment:
Do not cache the entire config in memory.
Instead, read the value you want and cache it there.